.github/workflows/ci.yml

```diff
@@ -0,0 +1,102 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}} | Cowboy ${{matrix.cowboy}})
+    strategy:
+      matrix:
+        elixir: ['1.7.4', '1.8.2', '1.9.4', '1.10.2']
+        cowboy: ['1', '2']
```

Comment:
I don't believe the `elixir` and `cowboy` settings in `matrix` does anything unless you also set the otp version. But they are not necessary since you also explicitly listed all versions in `include`.

Reply:
Correct. I set the matrix values to match what's currently in Travis CI config. This was the only way I could configure Github Actions to test the specific versions for parity with Travis.

Reply:
Yes, but you are duplicating the matrix values in the include listing. You don't need both.

Reply:
@ericmj Thanks for that. I don't remember Actions working that way originally but I'm glad it does now nonetheless.